tests/test_writer.py

```diff
@@ -191,3 +191,22 @@ def test_write_multiple_unknown_tag_diff_type():
     assert lines[6] == "JP  - CRISPR"
     assert lines[7] == "ED  - Swinburne, Ricardo"
     assert len(lines) == 9
+
+
+def test_delimited_dump():
+    entries = [
```

Comment:
can you update this test to add a multiple field that's not delimited to, to ensure it has one per line?